### ISSUE TITLE
Doesn't work on Django 3, and the fix

### ISSUE THREAD

**Issue by bharadwaj-raju** [ahmedaljazzar/django-mako#10]
Thanks for making django-mako!

Django 3 changed the location of an import, so django-mako fails with an ImportError.

changing [line 14](https://github.com/ahmedaljazzar/django-mako/blob/3a4099e8ae679f431eeb2c35024b2e6028f8a096/djangomako/backends.py#L14) in backends.py

from:

```py
from django.contrib.staticfiles.templatetags.staticfiles import static
```

to:

```py
from django.templatetags.static import static
```

makes it work.

**Comment by iamjazzar** [comment]
Fixed in #11, sorry for the long response. Thanks @emZubair

**Comment by emZubair** [comment]
> Fixed in #11, sorry for the long response. Thanks @emZubair

@iamjazzar no worries, can you please update the package on pypi?

**Comment by iamjazzar** [comment]
@emZubair Already released ([1.3.0](https://github.com/iamjazzar/django-mako/releases/tag/1.3.0)) and pushed on [pypi](https://pypi.org/project/djangomako/).
